# SingleCell-RNAseq-Clustering
Clustering Algorithms using Single-Cell RNA-Seq Data
Authors: Sarah Baalbaki, Sarah Oladejo, Sumitra Lele 

Single-cell RNA-seq data can be used to answer crucial questions about the differences between individual cells within an environment, and cells across different subtypes, which helps identify sub-populations and areas of interest for a specific problem being tackled. Efficient computational analyses of scRNA-seq data are required to extract inherent biological information from the huge quantity of information that single-cell RNA-Seq technology presents. The integration of single-cell RNA-seq data with other machine learning algorithms has facilitaed these methods of analysis. One of these algorithms is clustering which is applied on such data. It is an upstream step that affects other downstream analysis, and it is therefore imperative to obtain accurate cluster assignment and number of cell types from scRNA-Seq data.

In our project, we compare algorithms that use deep learning for clustering. We apply the algo- rithms to scRNA-Seq data and compare the different clustering methods and obtained results across different tissue types. The algorithms are benchmarked based on how much they deviate from the true cell type label provided. We also infer the correctness of the algorithm based on the differential expression of established marker genes in those tissue types.

